packages/mysql-community-8.0-mroonga/apt/debian-bookworm/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG FROM=debian:bookworm
+FROM ${FROM}
+
+RUN \
+  echo "debconf debconf/frontend select Noninteractive" | \
+    debconf-set-selections
+
+ENV \
+  DEBIAN_FRONTEND=noninteractive \
+  MYSQL_SERVER_VERSION=mysql-8.0
+
+ARG DEBUG
+
+RUN \
+  quiet=$([ "${DEBUG}" = "yes" ] || echo "-qq") && \
+  apt update ${quiet} && \
+  apt install -y -V --no-install-recommends ${quiet} \
+    ca-certificates \
+    lsb-release \
+    wget && \
+  wget https://packages.groonga.org/$(lsb_release --id --short | tr 'A-Z' 'a-z')/groonga-apt-source-latest-$(lsb_release --codename --short).deb && \
+  wget https://repo.mysql.com/mysql-apt-config_0.8.22-1_all.deb && \
```

Comment:
It seems that 0.8.25 is the latest: https://repo.mysql.com/

Reply:
Thank you for your comment.
The latest version of mysql-apt-config is 0.8.24-1 when I confirm https://repo.mysql.com/.
Therefore, I'll use 0.8.24-1.

Reply:
I've fixed it.

Reply:
Really? https://repo.mysql.com/mysql-apt-config_0.8.25-1_all.deb

Reply:
Oh...
https://repo.mysql.com/mysql-apt-config_0.8.25-1_all.deb can not show in https://repo.mysql.com/ on my web browser...

However, I can download this file with specify https://repo.mysql.com/mysql-apt-config_0.8.25-1_all.deb directly.
I'll use mysql-apt-config_0.8.25-1_all.deb.